person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  def initialize(_id, name, _age)
```

Comment:
- [ ] The Person class should have a `parent_permission` as optional and have a default value of `true`.
- [ ] The `name` parameter. should be optional and have a default value of `unknown`, kindly make sure to include these parameters so all the methods are useful